app/views/content/non-uk-teachers/find-a-teaching-job-as-a-non-uk-citizen.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Find a teaching job as a non-UK citizen"
+subcategory: If you want to train to teach
+description: |-
+  Learn more about applying for teaching jobs as a non-UK citizen after you qualify as a teacher.
+related_content:
+    Train to teach in England as an international student: "/non-uk-teachers/train-to-teach-in-england-as-an-international-student"
+    Teacher pay and benefits in England : "/is-teaching-right-for-me/teacher-pay-and-benefits"
+    A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
+promo_content:
+navigation: 20.60
+navigation_title: Find a teaching job as a non-UK citizen
+navigation_description: How to successfully apply for your first teaching job.
+date: "2021-05-27"
+image: "static/images/content/hero-images/0002.jpg"
+backlink: "../../"
+keywords:
+  - International
+  - Teaching
+  - Education
+  - Schools
+  - Teacher
+  - Overseas
+  - Foreign
+  - Non-UK
+  - Visa
+  - Skilled Worker visa
+  - Immigration
+  - Student Visa
+  - Graduate Visa
+  - Youth Mobility Scheme
+  - Family Visa
+  - UK Ancestry Visa
+  - Refugees
+  - Settlement Scheme
+  
+---
+
+English [qualified teacher status](/train-to-be-a-teacher/what-is-qts) allows you to teach in any type of school in England.
+
+Depending on your immigration status, you may have to [apply for a new visa](/non-uk-teachers/visas-for-non-uk-teachers) (usually a Skilled worker visa) to work as a teacher in the UK. 
+
+## Apply for your teaching job  
+
+To find a teaching job, you must apply to individual schools, not the Department for Education. If you need a Skilled Worker visa, you must find a school that can sponsor your visa. 
```

Comment:
I think this page used to say something about how schools don't accept CVs - we get a lot of feedback with internationals asking to upload their CVs, so might be worth adding back in?

Reply:
So I think legally, all schools cannot accept CVs - candidates must fill out an application either via Teaching Vacancies, or the school's website, or whatever other application method the school chooses (like the application forms of one of our competitors :( ). But it can't be a CV.

So if you just tweak to something like 'Schools cannot accept CVs so you will have to fill out an application for each role.'